js/client-chat-tournament.js

```diff
@@ -277,6 +277,7 @@
 						if (!this.info.isJoined)
 							this.toggleBoxVisibility(false);
 						this.room.$chat.append("<div class=\"notice tournament-message-start\">The tournament has started!</div>");
+						if (this.info.format.substr(0, 4) === 'gen5' && !Tools.loadedSpriteData['bw']) Tools.loadSpriteData('bw');
 						break;
 
```

Comment:
Or once people start switching in the first game. It's not ideal, but it's better than nothing.